hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/AbstractHoodieWriteClient.java

```diff
@@ -989,12 +989,14 @@ protected void rollbackFailedWrites(Map<String, Option<HoodiePendingRollbackInfo
   /**
    * Performs a compaction operation on a table, serially before or after an insert/upsert action.
    */
-  protected Option<String> inlineCompact(Option<Map<String, String>> extraMetadata) {
+  protected Option<String> inlineCompact(Option<Map<String, String>> extraMetadata, boolean executeInline) {
     Option<String> compactionInstantTimeOpt = scheduleCompaction(extraMetadata);
-    compactionInstantTimeOpt.ifPresent(compactInstantTime -> {
```

Comment:
The method name is `inlineCompact` and why is still a flag `executeInline` there ?